sickbeard/name_parser/parser.py

```diff
@@ -346,19 +347,21 @@ def _combine_results(self, first, second, attr):
         b = getattr(second, attr)
 
         # if a is good use it
-        if a != None or (type(a) == list and len(a)):
+        if a != None or (isinstance(a, list) and a):
```

Comment:
Can't this just be `a != None`?
If `a` is `None`, the first part is `False` and the second part is `False` too (`None` is not a `list`).
If `a` is not `None`, the first part is `True` and the rest is not evaluated.